### PR TITLE
Debug guards for sbp logging

### PR DESCRIPTION
**Pull request by anth-cole**
RELEASE PR for this issue https://github.com/swift-nav/piksi_v3_bug_tracking/issues/424

### CODE REVIEW

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.15-2-g35e7c9a
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.15-2-g35e7c9a/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.15-2-g35e7c9a/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.15-2-g35e7c9a

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.15-3-g340ae77
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.15-3-g340ae77/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.15-3-g340ae77/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.15-3-g340ae77

File: package/sbp_rtcm3_bridge/src/sbp_rtcm3.h [hunk 1]
```diff
@@ -28,6 +29,8 @@
 #define MAX_SBP_PAYLOAD 255
 #define MAX_OBS_IN_SBP ((MAX_SBP_PAYLOAD - SBP_HEADER_SIZE) / SBP_OBS_SIZE)
 
+static bool debug = false;
```
Comment (mfine) [inline package/sbp_rtcm3_bridge/src/sbp_rtcm3.h]: Does this work? Won't it just put a version of debug in each compilation unit - I think this needs to be extern'd.

**Comment by anonymous-travis-swiftnav** [comment]
## v1.1.15-6-geea77d9
+ [Artifacts (HITL Dashboard Page)](http://hitl-dashboard.swiftnav.com/files/swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.15-6-geea77d9/)
+ [Artifacts (S3 / AWS Console)](https://console.aws.amazon.com/s3/home?region=us-west-2&bucket=swiftnav-artifacts-pull-requests&prefix=piksi_buildroot/v1.1.15-6-geea77d9/)
+ s3://swiftnav-artifacts-pull-requests/piksi_buildroot/v1.1.15-6-geea77d9

**Pull request merged**